changelogs/fragments/4168-add-keycloak-url-timeout.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "Keycloak module - added connection timeout parameter when calling server (https://github.com/ansible-collections/community.general/pull/4168)."
```

Comment:
```suggestion
  - "keycloak_* modules - added connection timeout parameter when calling server (https://github.com/ansible-collections/community.general/pull/4168)."
```

Reply:
Oh, ok, that's why it was complaining! Thanks a lot!